### PR TITLE
Update mei-board members for 2023

### PR DESCRIPTION
**Pull request by bwbohl**
This is still a draft, especially as @annplaksin and @musicog still should review their affiliations, AND of course it should only be merged after 2022

### CODE REVIEW

**Review by musicog — CHANGES_REQUESTED** [review]
Very exciting! :-)

Two small modification requests: Please flip the Institut in front of the Universität, and please add "mdw –" before the Universität (...branding reasons...), like so:

> **David M. Weigl**, Institut für musikalische Akustik – Wiener Klangstil, mdw – Universität für Musik und darstellende Kunst Wien

Thanks!

**Review by annplaksin — APPROVED** [review]
So exciting!
I'm fine, with the changes

**Review by rettinghaus — APPROVED** [review]

**Review by musicog — APPROVED** [review]
Perfect, thanks!

**Comment by rettinghaus** [comment]
@musicEnfanthen please merge if appropriate.

File: community/mei-board.md [hunk 1]
```diff
@@ -22,14 +28,10 @@ The MEI Board manages the property and business of MEI. It promotes the developm
...
-### Term 2020–2022
-
-* **Johannes Kepper**, Musikwissenschaftliches Seminar Detmold/Paderborn
-* **Perry Roland**, University of Virginia Library, Charlottesville (previous terms 2015–2018)
-* **Martha Thomae**, McGill University, Montréal
-
 ### Previous Board Members
 
+* **Martha Thomae**, McGill University, Montréal (2020–2022)
+* **Perry Roland**, University of Virginia Library, Charlottesville (2015–2018, 2020–2022)
```
Comment (musicEnfanthen) [inline community/mei-board.md:34]: If I remember correctly, we used alphabetical order for previous Board members. @bwbohl Can you please sort accordingly?

**Review by rettinghaus — APPROVED** [review]

**Review by musicEnfanthen — APPROVED** [review]
Thank you

**Pull request merged**